### ISSUE TITLE
Proposal for implementing a style guide in the framework and examples

### ISSUE THREAD

**Issue by a-r-r-o-w** [uptrain-ai/uptrain#136]
Currently, it isn't enforced to follow a style guide (such as pep8) when working on the framework or examples. This could lead to inconsistent formatting across files as different contributors get involved. I propose using an automated code formatter tool that will make it easy for new contributors to format their code as expected in the style-guide as well as format current non-style-guide compliant code.

After looking at multiple other open-source repositories, the leading automated code formatter seems to be [Black](https://github.com/psf/black). Projects also usually add it to their DevOps pipeline to automate the process on every new code contribution.

Black has very simple usage:

```
# run as script
black {source_file_or_directory}

# or run as package
python3 -m black {source_file_or_directory}
```

To encourage new contributors to adhere to the style-guide, I propose adding instructions to run Black in the [CONTRIBUTING.md](https://github.com/uptrain-ai/uptrain/blob/main/CONTRIBUTING.md) file. Furthermore, the tool can be used to format the existing code base to ensure consistency in the formatting.

Overall, integrating Black into the UpTrain framework and examples would improve the consistency and readability of the codebase, while simplifying the code formatting process for all contributors.

**Comment by vipgupta** [comment]
Hi @a-r-r-o-w , thanks for the suggestion, very much appreciated. We have been using BLACK, but not consistently. Can you add the relevant part in contributing.md?

**Comment by vipgupta** [comment]
Closed by #140
